From 8132c99f5e03594aa727990183a1353cb3323a26 Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Mon, 20 Aug 2018 22:57:43 +0300
Subject: [PATCH 204/296] modify overscroll bug in
 NotificationStackScrollLayout After overscroll,

notifiation panel is seen as being cut. Initialize overscroll when you unlock
---
 .../systemui/statusbar/stack/NotificationStackScrollLayout.java        | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationStackScrollLayout.java b/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationStackScrollLayout.java
index eb263906328..7daa2ebc98f 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationStackScrollLayout.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationStackScrollLayout.java
@@ -4284,6 +4284,9 @@ public class NotificationStackScrollLayout extends ViewGroup
     }
 
     public void setStatusBarState(int statusBarState) {
+        if (mStatusBarState != statusBarState) {
+            endDrag();
+        }
         mStatusBarState = statusBarState;
         mAmbientState.setStatusBarState(statusBarState);
     }
-- 
2.11.0

